index.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+app.use(express.json())
+
+app.get('/', (req, res) => {
+    res.status(200);
+    res.json({message: 'Welcome to the root URL of the server'})
```

Comment:
Move your controller functions to a separate file and add middleware for authentication and authorization.